src/libs/blueprint/conduit_blueprint_mesh.hpp

```diff
@@ -291,6 +333,11 @@ namespace topology
         //---------------------------------------------------------------------
         bool CONDUIT_BLUEPRINT_API verify(const conduit::Node &n,
                                           conduit::Node &info);
+
+        //---------------------------------------------------------------------
+        bool CONDUIT_BLUEPRINT_API transform(const conduit::Node &n,
+                                             conduit::Node &dest,
+                                             conduit::Node &cdest);
     }
```

Comment:
did we miss one `transform`?

Reply:
Good catch! This extraneous function declaration has now been removed.